src/components/Datasets.js

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { withRouter } from 'react-router';
+import { darken, makeStyles } from '@material-ui/core/styles';
+
+import Container from '@material-ui/core/Container';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import IconButton from '@material-ui/core/IconButton';
+import PublishIcon from '@material-ui/icons/Publish';
+import EqualizerIcon from '@material-ui/icons/Equalizer';
+import EditIcon from '@material-ui/icons/Edit';
+import DeleteIcon from '@material-ui/icons/Delete';
+import AddIcon from '@material-ui/icons/Add';
+
+import theme from '../theme';
+import Main from './common/Main';
+
+const fakeDatasets = [
+  {
+    name: 'Atomic Structure',
+    description: 'Atomic Structure Dataset of students during two months',
+    size: 413,
+    created: '7/13/20',
+    lastModified: '7/13/20',
+    anonymized: false,
+  },
+  {
+    name: 'Animals of the world',
+    description: 'Collection of all the referenced animals of world.',
+    size: 409600,
+    created: '7/12/18',
+    lastModified: '9/14/20',
+    anonymized: false,
+  },
+  {
+    name: 'Anonymized Atomic Structure',
+    description: 'Anonymized version of Atomic Structure Dataset',
+    size: 211,
+    created: '7/02/20',
+    lastModified: '7/02/20',
+    anonymized: true,
+  },
+];
+
+function descendingComparator(a, b, orderBy) {
```

Comment:
In ES6, we can use arrow functions https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions It changes a how `this` is handled, but in this case it is not very important, but you might change it.